src/eko/mellin.py

```diff
@@ -222,8 +222,8 @@ class Path:
 
     def __init__(self, t, logx, axis_offset):
         self.t = t
-        # TODO: shall we use: 0.4 * 16.0 / (1.0 - logx) ?
-        self.r = 0.4 * 16.0 / (-logx)
+        # TODO: shall we use: 0.4 * 16.0 / ( - logx) ?
+        self.r = 0.4 * 16.0 / (1.0 - logx)
```

Comment:
I guess this has been tested, are the benefits documented somewhere? It might be useful for the future.

Consider writing something either here in the docstring, or in the Mellin page of the docs.